### DISCUSSION TITLE
Select fields from through model

### DISCUSSION THREAD

**Discussion by Abdeldjalil-H** [collerek/ormar#872] [Q&A] [answered]
I have three models
```python
class UserAddress(Model):
  phone: str

class Address(Model):
  ...

class User(Model):
  # ...
  addresses = ManyToMany(Address, through=UserAddress)
```
I want to get all user addresses with the phone number for each address. For that I need to fetch the `UserAddress` for each address, I tried the following
```python
user.addresses.select_related("useraddress")
```
I get the following exception
```
ormar.exceptions.RelationshipInstanceError: Alias for given relation not found.
```

**Comment by collerek** [comment, marked as answer]
It's described in the docs: https://collerek.github.io/ormar/relations/many-to-many/#through-fields
